hwdb/70-pointingstick.hwdb

```diff
@@ -142,3 +142,7 @@ evdev:name:TPPS/2 IBM TrackPoint:dmi:bvn*:bvr*:bd*:svnLENOVO:pn*:pvrThinkPadX20?
 evdev:input:b0003v06CBp0009*
  POINTINGSTICK_CONST_ACCEL=2.5
  POINTINGSTICK_SENSITIVITY=200
+
+# ThinkPad X280 with an ALPS trackpoint
+evdev:name:*ALPS TrackPoint*:svnLENOVO:*:pvrThinkPadX280:*
+ POINTINGSTICK_SENSITIVITY=64
```

Comment:
As mentioned on IRC, the header says `"Entries should be sorted with growing _SENSITIVITY and _CONST_ACCEL."` so this should be at the top of the section.